### DISCUSSION TITLE
Do not send request to glyphs and not working with plugin: "maplibre-gl-measures"

### DISCUSSION THREAD

**Discussion by SO-Matveev** [maplibre/maplibre-gl-js#4471] [Q&A]
I have demo project from Angular. I create one page and one components map-layout for maplibre-gl-js. Then i install plugin maplibre-gl-measures. This is my code from example
````
 import MeasuresControl from 'maplibre-gl-measures'

@Component({
  selector: "app-map-layout",
  templateUrl: "./map-layout.component.html",
  standalone: true,

  styleUrls: ["./map-layout.component.css"]
})

export class MapLayoutComponent implements OnInit{
  ngOnInit(): void {
    const map: maplibregl.Map = new maplibregl.Map({
      container: 'map', // container id
      style: {
        version: 8,
        sources: {
          osm: {
            type: "raster",
            tiles: ["https://a.tile.openstreetmap.org/{z}/{x}/{y}.png"],
            tileSize: 256,
            attribution: "&copy; OpenStreetMap Contributors",
            maxzoom: 19
          }
        },
        glyphs: "https://fonts.openmaptiles.org/{fontstack}/{range}.pbf",
        layers: [
          {
            id: "osm",
            type: "raster",
            source: "osm"
          }
        ]
      },
      center: [-8, 40], // starting position [lng, lat]
      zoom: 15 // starting zoom
    });
    console.log('map', map)
    map.addControl(new MeasuresControl(), "top-left");

  }
}
````
Then i added new control from maplibre-gl-measures and he partially works.
I see button, he paint lines or polygon but i not see total area or length.
![map](https://github.com/user-attachments/assets/40e41a31-d1ea-47a5-ac2b-123514410d16)

The mistake is that maplibre-gl does not send request to glyphs at the specified address as in the example.

Please, help!
What am I doing wrong?

I tried to use local glyphs. Also not worked.
Unfortunately maplibre-gl not used glyphs, but when open example it`s working
https://github.com/jdsantos/maplibre-gl-measures
https://jdsantos.github.io/demos/maplibre-gl-measures/

**Comment by HarelM** [comment]
Try to update the configuration of the control to use different font that know are working.
If this doesn't help I would recommend opening an issue in the measures repo instead of here.

**Reply by SO-Matveev** [reply]
Thank you for your reply.
The issue is already open in measures repo.
